staging/src/github.com/kubeedge/viaduct/pkg/packer/reader.go

```diff
@@ -7,6 +7,10 @@ import (
 	"k8s.io/klog/v2"
 )
 
+const (
+	maxPayloadLen = 32 * 1 << 20 // 32Mi
```

Comment:
What is the basis of this value？

Reply:
This type of limit in k8s is generally 10MB, but I feel that this scenario may have a configuration file exceeding 10MB, so increase it to 32MB